Assets/Resources/UI/DB_Trade.prefab

```diff
@@ -48,7 +48,7 @@ GameObject:
   m_Icon: {fileID: 0}
   m_NavMeshLayer: 0
   m_StaticEditorFlags: 0
-  m_IsActive: 0
+  m_IsActive: 1
```

Comment:
This is may be why the trade dialog box pop up at the begining of a game

You have to disable it in the editor before saving the prefab (checkbox enable uncheck)

![image](https://cloud.githubusercontent.com/assets/6545998/18409858/898fcb1e-7752-11e6-9989-786203bff292.png)